gax/src/main/java/com/google/api/gax/batching/BatcherImpl.java

```diff
@@ -224,6 +291,10 @@ public void close() throws InterruptedException {
     }
   }
 
+  FlowController getFlowController() {
```

Comment:
Why is this method needed?

Reply:
This is for testing, added `visibleForTesting` annotation and documentation.